src/rgw/rgw_rest_s3.h

```diff
@@ -59,6 +59,33 @@ class RGWGetObj_ObjStore_S3 : public RGWGetObj_ObjStore
                          bufferlist* manifest_bl) override;
 };
 
+class RGWGetObjTags_ObjStore_S3 : public RGWGetObjTags_ObjStore
+{
+  bufferlist tags_bl;
```

Comment:
duplicated with RGWGetObjTags::tags_bl

Reply:
thanks @Liuchang0812, could you open a pr to remove this one?

Reply:
Of course. I will do it